cloudapi/api.go

```diff
@@ -336,3 +338,95 @@ func (c *Client) ValidateToken() (*ValidateTokenResponse, error) {
 
 	return &vtr, nil
 }
+
+// Organization represents a Grafana Cloud k6 organization
+type Organization struct {
+	ID               int    `json:"id"`
+	GrafanaStackName string `json:"grafana_stack_name"`
+	GrafanaStackID   int    `json:"grafana_stack_id"`
+}
+
+// AccountMeResponse represents the response of the /account/me endpoint.
+type AccountMeResponse struct {
+	Organizations []Organization `json:"organizations"`
+}
+
+// AccountMe retrieves the current user's account information.
+func (c *Client) AccountMe() (*AccountMeResponse, error) {
+	// TODO: remove this hardcoded URL
+	req, err := c.NewRequest("GET", "https://api.k6.io/v3/account/me", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	amr := AccountMeResponse{}
+	err = c.Do(req, &amr)
+	if err != nil {
+		return nil, err
+	}
+
+	return &amr, nil
+}
+
+// GetDefaultProject retrieves the default project for the given stack ID.
+func (c *Client) GetDefaultProject(stackID int64) (int64, string, error) {
+	// TODO: remove this hardcoded URL
+	req, err := c.NewRequest("GET", "https://api.k6.io/cloud/v6/projects", nil)
+	if err != nil {
+		return 0, "", err
+	}
+
+	q := req.URL.Query()
+	q.Add("$orderby", "created")
+	req.URL.RawQuery = q.Encode()
+
+	req.Header.Set("X-Stack-Id", fmt.Sprintf("%d", stackID))
+	// TODO: by default the client uses Token instead of Bearer
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.token))
+	req.Header.Set("User-Agent", "Go-http-client")
+
+	// TODO: Can't use c.Do bc it messes up the headers
```

Comment:
I think this is yet another indicator that we should likely have a different client implementation for this.